components/Dates/DateForm.js

```diff
@@ -105,9 +133,24 @@ export default function DateForm({ locationId }) {
         rows="5"
         maxLength={100}
       ></Textarea>
-      <SubmitButton type="submit" backgroundcolor="globalPlantBackgroundColor">
+      {!savedStatus ? (
+        <SubmitButton
+          type="submit"
+          backgroundcolor="globalPlantBackgroundColor"
+        >
+          Jetzt Speichern
+        </SubmitButton>
+      ) : (
+        <SubmitButton
+          type="submit"
+          backgroundcolor="globalPlaceBackgroundColor"
+        >
+          Gespeichert
+        </SubmitButton>
+      )}
+      {/* <SubmitButton type="submit" backgroundcolor="globalPlantBackgroundColor">
         Speichern
-      </SubmitButton>
+      </SubmitButton> */}
```

Comment:
You can remove this if it's not doing anything :)

Reply:
Thanks for the tip. I usually do that before I create a PR. I'm still working on the components, so such code snippets stay a bit in their place.

Reply:
✅